erica/infrastructure/sqlalchemy/database.py

```diff
@@ -7,11 +9,17 @@
 from erica.config import get_settings
 
 
+def default(obj):
```

Comment:
Why call that default? And why raise a TypeError if it is not a decimal?

I think I am a little confused 😬

Reply:
This is the standard proposed way from orjson when we have data types that need to be serialized but orjson does not natively support it. And Decimal is one of those. This default function is then only called if orjson cannot serializable the data type, meaning if we get a str, the default function won't be called at all.

So raising a TypeError (as shown [here](https://github.com/ijl/orjson#default)) would just be the same as not implementing a specific converter for a data type not natively supported by orjson.